Dockerfile

```diff
@@ -17,8 +17,13 @@ RUN dnf install -y \
     util-linux-user \
     veritysetup \
     xz \
-    zypper
+    zypper \
+# Add some necesary packages to fix bugs:
+    e2fsprogs \
```

Comment:
Although I naturally try to keep dependencies to a minimum, it's not a big issue with this project.
I do think these are nice additions for building images. I also needed systemd-container at one point. So I will add them to the .